webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/servers/RemoveNodeService.scala

```diff
@@ -303,11 +316,164 @@ class RemoveNodeServiceImpl(
   }
 
 
-  private[this] def atomicDelete(nodeId : NodeId, modId: ModificationId, actor:EventActor) : IOResult[Seq[LDIFChangeRecord]] = {
+  ////////////////////////////////
+  //// implementation details ////
+  ////////////////////////////////
+
+
+  // find the status of node to delete. It can have multiple result if inventories exists in several status (ex: pending and deleted)
+  def findNodeStatus(nodeId: NodeId): IOResult[Set[InventoryStatus]] = {
```

Comment:
Should be plural, I expected an option (since it was at least an iterable) with this name